src/main/java/de/metas/ui/web/product/process/WEBUI_M_Product_LotNumber_Lock.java

```diff
@@ -115,39 +144,22 @@ private void quarantineHUsForLotNo(final int lotNoLockId)
 			final int bpartnerId = firstReceipt.getC_BPartner_ID();
 			final int bpLocationId = firstReceipt.getC_BPartner_Location_ID();
 
-			husToDistribute.add(HUToDistribute.builder()
-					.hu(hu)
-					.lockLotNo(lotNoLock)
-					.bpartnerId(bpartnerId)
-					.bpartnerLocationId(bpLocationId)
-					.build());
+			husToDistribute.add(HUToDistribute.builder().hu(hu)
+					.lockLotNo(lotNoLock).bpartnerId(bpartnerId)
+					.bpartnerLocationId(bpLocationId).build());
 		}
 
-		Services.get(IHUDDOrderBL.class).createQuarantineDDOrderForHUs(husToDistribute);
-
-		setExistingInvoiceCandsInDispute(husForAttributeStringValue);
-
 	}
 
-	private void setExistingInvoiceCandsInDispute(final List<I_M_HU> hus)
-	{
-
-		hus.stream()
-				.flatMap(hu -> huInOutDAO.retrieveInOutLinesForHU(hu).stream())
-				.forEach(line -> {
-					invoiceCandBL.markInvoiceCandInDisputeForReceiptLine(line);
-				});
-
-	}
-
-	private List<I_M_HU> retrieveHUsForAttributeStringValue(final int productId, final I_M_Attribute attribute, final String value)
+	private List<I_M_HU> retrieveHUsForAttributeStringValue(final int productId,
+			final I_M_Attribute attribute, final String value)
```

Comment:
in case you have the parameters on multiple lines, make sure that:
* on first line it's only the method (i.e. `final int productId` shall go to next line`
* you don't have more than one parameter per line (i.e. `final String value` shall go to it's own line)
* the closing parenthesis  shall be on the same line with last parameter (i.e. `final String value)`)

Reply:
Done. Thanks!